MidCamp/midcamp
# Url aliases

Add basic url aliases. (We can always tweak it later).

To test
- `composer install`
- create content types and observe they're aliased
- create event terms (with hierarchy) and observe fun aliasing
- create users and obser aliases
